src/accessibility.cs

```diff
@@ -53,4 +140,173 @@ interface AXCustomContentProvider
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXDataPointValue : NSCopying
+	{
+		[Export ("number", ArgumentSemantic.Assign)]
+		double Number { get; set; }
+
+		[Export ("category")]
+		string Category { get; set; }
+
+		[Static]
+		[Export ("valueWithNumber:")]
+		AXDataPointValue ValueWithNumber (double number);
+
+		[Static]
+		[Export ("valueWithCategory:")]
+		AXDataPointValue ValueWithCategory (string category);
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXDataSeriesDescriptor : NSCopying
+	{
+		[NullAllowed, Export ("name")]
+		string Name { get; set; }
+
+		[Export ("attributedName", ArgumentSemantic.Copy)]
+		NSAttributedString AttributedName { get; set; }
+
+		[Export ("isContinuous")]
+		bool IsContinuous { get; set; }
+
+		[Export ("dataPoints", ArgumentSemantic.Copy)]
+		AXDataPoint[] DataPoints { get; set; }
+
+		[Export ("initWithName:isContinuous:dataPoints:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string name, bool isContinuous, AXDataPoint[] dataPoints);
+
+		[Export ("initWithAttributedName:isContinuous:dataPoints:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSAttributedString attributedName, bool isContinuous, AXDataPoint[] dataPoints);
+	}
+
+	[Watch (8,0), NoTV, NoMac, iOS (15,0)]
+	[Flags]
+	[Native]
+	public enum AXHearingDeviceEar : ulong
+	{
+		None = 0,
+		Left = 1 << 1,
+		Right = 1 << 2,
+		Both = Left | Right,
+	}
+
+	[Watch (8,0), NoTV, NoMac, iOS (15,0)]
+	[Static]
+	[Partial]
+	partial interface AXHearingUtilities
+	{
+		[Field ("AXMFiHearingDeviceStreamingEarDidChangeNotification")]
+		NSString StreamingEarDidChangeNotification { get; }
+
+		[Field ("AXMFiHearingDevicePairedUUIDsDidChangeNotification")]
+		NSString PairedUUIDsDidChangeNotification { get; }
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	interface AXLiveAudioGraph
+	{
+		[Static]
+		[Export ("start")]
+		void Start ();
+
+		[Static]
+		[Export ("updateValue:")]
+		void UpdateValue (double value);
+
+		[Static]
+		[Export ("stop")]
+		void Stop ();
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[Native]
+	public enum AXNumericDataAxisDescriptorScale : long
+	{
+		linear = 0,
+		log10,
+		ln,
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXNumericDataAxisDescriptor : AXDataAxisDescriptor
+	{
+		[Export ("scaleType", ArgumentSemantic.Assign)]
+		AXNumericDataAxisDescriptorScale ScaleType { get; set; }
+
+		[Export ("lowerBound")]
+		double LowerBound { get; set; }
+
+		[Export ("upperBound")]
+		double UpperBound { get; set; }
+
+		[Export ("valueDescriptionProvider", ArgumentSemantic.Copy)]
+		Func<double, NSString> ValueDescriptionProvider { get; set; }
```

Comment:
Maybe a delegate will be more helpful for the developers to understand what is going on.

Reply:
If I'm understanding correctly, would that be something like this? 

```cs
delegate double AXDataValue;
delegate NSString AXDataValueDescription;

[Export ("valueDescriptionProvider", ArgumentSemantic.Copy)]
Func<AXDataValue, AXDataValueDescription> ValueDescriptionProvider { get; set; }
```

if not, what were you thinking?

Reply:
No, more replacing `Func` with a `delegate` type. The later are self-documenting. 

The more arguments the more important it becomes because `Func<byte,short,int,long>` has to meaning (without reading the docs). However a delegate type like `delegate long Hash (byte b, short s, int i);` describe what the function is expected to do.

Reply:
Something like:
```shaper
delegate AXDataValueDescription ValueDescriptionProviderHandler (AXDataValue dataValue);
```
To be honest, the headers do not help much.